+ source ../common/common_lib.sh
++ source /root/mugen/libs/locallibs/common_lib.sh
+++ python3 --version
+++ '[' 0 -eq 0 ']'
+++ source /root/mugen/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start environmental preparation.'
+ message='Start environmental preparation.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start environmental preparation.'
Wed Apr  5 16:32:43 2023 - INFO  - Start environmental preparation.
+ cp /etc/dhcp/dhcpd6.conf /etc/dhcp/dhcpd6.bak
cp: cannot stat '/etc/dhcp/dhcpd6.conf': No such file or directory
+ echo 'default-lease-time 2592000;
preferred-lifetime 604800;
option dhcp-renewal-time 3600;
option dhcp-rebinding-time 7200;
allow leasequery;
option dhcp6.info-refresh-time 21600;
subnet6 2001:db8::0/64 {
    range6 2001:db8::50 2001:db8::100;
}'
+ ip addr add 2001:db8::1 dev ''
Cannot find device ""
+ LOG_INFO 'End of environmental preparation!'
+ message='End of environmental preparation!'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End of environmental preparation!'
Wed Apr  5 16:32:44 2023 - INFO  - End of environmental preparation!
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start testing...'
+ message='Start testing...'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start testing...'
Wed Apr  5 16:32:45 2023 - INFO  - Start testing...
+ test_execution dhcpd6.service
+ service=dhcpd6.service
++ date '+%Y-%m-%d %T'
+ log_time='2023-04-05 16:32:45'
+ test_restart dhcpd6.service
+ service=dhcpd6.service
+ systemctl restart dhcpd6.service
Failed to restart dhcpd6.service: Unit dhcpd6.service not found.
+ CHECK_RESULT 5 0 0 'dhcpd6.service restart failed'
+ actual_result=5
+ expect_result=0
+ mode=0
+ error_log='dhcpd6.service restart failed'
+ exit_mode=0
+ '[' -z 5 ']'
+ '[' 0 -eq 0 ']'
+ test 5x '!=' 0x
+ test -n 'dhcpd6.service restart failed'
+ LOG_ERROR 'dhcpd6.service restart failed'
+ message='dhcpd6.service restart failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'dhcpd6.service restart failed'
Wed Apr  5 16:32:46 2023 - ERROR - dhcpd6.service restart failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 34'
+ message='../common/common_lib.sh line 34'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 34'
Wed Apr  5 16:32:47 2023 - ERROR - ../common/common_lib.sh line 34
+ '[' 0 -eq 1 ']'
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status dhcpd6.service
+ grep 'Active: active'
Unit dhcpd6.service could not be found.
+ CHECK_RESULT 1 0 0 'dhcpd6.service restart failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='dhcpd6.service restart failed'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'dhcpd6.service restart failed'
+ LOG_ERROR 'dhcpd6.service restart failed'
+ message='dhcpd6.service restart failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'dhcpd6.service restart failed'
Wed Apr  5 16:32:55 2023 - ERROR - dhcpd6.service restart failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 37'
+ message='../common/common_lib.sh line 37'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 37'
Wed Apr  5 16:32:56 2023 - ERROR - ../common/common_lib.sh line 37
+ '[' 0 -eq 1 ']'
+ return 0
+ systemctl stop dhcpd6.service
Failed to stop dhcpd6.service: Unit dhcpd6.service not loaded.
+ CHECK_RESULT 5 0 0 'dhcpd6.service stop failed'
+ actual_result=5
+ expect_result=0
+ mode=0
+ error_log='dhcpd6.service stop failed'
+ exit_mode=0
+ '[' -z 5 ']'
+ '[' 0 -eq 0 ']'
+ test 5x '!=' 0x
+ test -n 'dhcpd6.service stop failed'
+ LOG_ERROR 'dhcpd6.service stop failed'
+ message='dhcpd6.service stop failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'dhcpd6.service stop failed'
Wed Apr  5 16:32:57 2023 - ERROR - dhcpd6.service stop failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 39'
+ message='../common/common_lib.sh line 39'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 39'
Wed Apr  5 16:32:58 2023 - ERROR - ../common/common_lib.sh line 39
+ '[' 0 -eq 1 ']'
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status dhcpd6.service
+ grep 'Active: inactive'
Unit dhcpd6.service could not be found.
+ CHECK_RESULT 1 0 0 'dhcpd6.service stop failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='dhcpd6.service stop failed'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'dhcpd6.service stop failed'
+ LOG_ERROR 'dhcpd6.service stop failed'
+ message='dhcpd6.service stop failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'dhcpd6.service stop failed'
Wed Apr  5 16:33:05 2023 - ERROR - dhcpd6.service stop failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 42'
+ message='../common/common_lib.sh line 42'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 42'
Wed Apr  5 16:33:06 2023 - ERROR - ../common/common_lib.sh line 42
+ '[' 0 -eq 1 ']'
+ return 0
+ systemctl start dhcpd6.service
Failed to start dhcpd6.service: Unit dhcpd6.service not found.
+ CHECK_RESULT 5 0 0 'dhcpd6.service start failed'
+ actual_result=5
+ expect_result=0
+ mode=0
+ error_log='dhcpd6.service start failed'
+ exit_mode=0
+ '[' -z 5 ']'
+ '[' 0 -eq 0 ']'
+ test 5x '!=' 0x
+ test -n 'dhcpd6.service start failed'
+ LOG_ERROR 'dhcpd6.service start failed'
+ message='dhcpd6.service start failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'dhcpd6.service start failed'
Wed Apr  5 16:33:07 2023 - ERROR - dhcpd6.service start failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 44'
+ message='../common/common_lib.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 44'
Wed Apr  5 16:33:08 2023 - ERROR - ../common/common_lib.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status dhcpd6.service
+ grep 'Active: active'
Unit dhcpd6.service could not be found.
+ CHECK_RESULT 1 0 0 'dhcpd6.service start failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='dhcpd6.service start failed'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'dhcpd6.service start failed'
+ LOG_ERROR 'dhcpd6.service start failed'
+ message='dhcpd6.service start failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'dhcpd6.service start failed'
Wed Apr  5 16:33:15 2023 - ERROR - dhcpd6.service start failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 47'
+ message='../common/common_lib.sh line 47'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 47'
Wed Apr  5 16:33:16 2023 - ERROR - ../common/common_lib.sh line 47
+ '[' 0 -eq 1 ']'
+ return 0
+ test_enabled dhcpd6.service
+ service=dhcpd6.service
++ systemctl is-enabled dhcpd6.service
Failed to get unit file state for dhcpd6.service: No such file or directory
+ state=
+ '[' '' == enabled ']'
+ '[' '' == disabled ']'
+ '[' '' == masked ']'
+ '[' '' == static ']'
+ LOG_INFO 'Unit is indirect, ignoring.'
+ message='Unit is indirect, ignoring.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Unit is indirect, ignoring.'
Wed Apr  5 16:33:17 2023 - INFO  - Unit is indirect, ignoring.
+ journalctl --since '2023-04-05 16:32:45' -u dhcpd6.service
+ grep -i 'fail\|error'
+ grep -v -i 'DEBUG\|INFO\|WARNING'
+ CHECK_RESULT 1 0 1 'There is an error message for the log of dhcpd6.service'
+ actual_result=1
+ expect_result=0
+ mode=1
+ error_log='There is an error message for the log of dhcpd6.service'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 1 -eq 0 ']'
+ test 1x == 0x
+ return 0
+ test_reload dhcpd6.service
+ service=dhcpd6.service
+ systemctl start dhcpd6.service
Failed to start dhcpd6.service: Unit dhcpd6.service not found.
+ systemctl reload dhcpd6.service
+ grep 'Job type reload is not applicable'
+ CHECK_RESULT 1 0 0 'Job type reload is not applicable for unit dhcpd6.service'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Job type reload is not applicable for unit dhcpd6.service'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Job type reload is not applicable for unit dhcpd6.service'
+ LOG_ERROR 'Job type reload is not applicable for unit dhcpd6.service'
+ message='Job type reload is not applicable for unit dhcpd6.service'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Job type reload is not applicable for unit dhcpd6.service'
Wed Apr  5 16:33:18 2023 - ERROR - Job type reload is not applicable for unit dhcpd6.service
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 91'
+ message='../common/common_lib.sh line 91'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 91'
Wed Apr  5 16:33:19 2023 - ERROR - ../common/common_lib.sh line 91
+ '[' 0 -eq 1 ']'
+ return 0
+ systemctl status dhcpd6.service
+ grep 'Active: active'
Unit dhcpd6.service could not be found.
+ systemctl status dhcpd6.service
+ grep 'inactive (dead)'
Unit dhcpd6.service could not be found.
+ return 1
+ LOG_INFO 'Finish test!'
+ message='Finish test!'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Finish test!'
Wed Apr  5 16:33:20 2023 - INFO  - Finish test!
+ CASE_RESULT 0
+ case_re=0
+ test -z 7
+ test 7 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Wed Apr  5 16:33:21 2023 - ERROR - failed to execute the case.
+ exit 7
+ post_test
+ LOG_INFO 'start environment cleanup.'
+ message='start environment cleanup.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'start environment cleanup.'
Wed Apr  5 16:33:22 2023 - INFO  - start environment cleanup.
+ systemctl stop dhcpd6.service
Failed to stop dhcpd6.service: Unit dhcpd6.service not loaded.
+ mv -f /etc/dhcp/dhcpd6.bak /etc/dhcp/dhcpd6.conf
mv: cannot stat '/etc/dhcp/dhcpd6.bak': No such file or directory
+ ip addr del 2001:db8::1 dev ''
Cannot find device ""
+ LOG_INFO 'Finish environment cleanup!'
+ message='Finish environment cleanup!'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Finish environment cleanup!'
Wed Apr  5 16:33:24 2023 - INFO  - Finish environment cleanup!
